crates/node/core/src/args/hardfork_overrides.rs

```diff
@@ -0,0 +1,230 @@
+use clap::Args;
+use reth_chainspec::ChainHardforks;
+use reth_ethereum_forks::{EthereumHardfork, ForkCondition};
+
+/// Command line arguments for overriding Ethereum hardfork configurations.
+///
+/// These arguments allow users to override the default timestamps for various
+/// Ethereum hardforks, enabling custom network configurations and testing scenarios.
+#[derive(Default, Debug, Args, Clone)]
+#[command(next_help_heading = "Hardfork Overrides")]
+pub struct HardforkOverridesArgs {
```

Comment:
I think we can reduce these to just prague+osaka, since this feature is intended for testing upcoming hardforks, there's no need to support older hardforks